client/src/components/Toast/Toast.js

```diff
@@ -0,0 +1,134 @@
+import React from 'react';
+
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import CheckCircleIcon from '@material-ui/icons/CheckCircle';
```

Comment:
It would be good to group some imports. For example:
import { AppBar, Toolbar, IconButton, InputBase, MenuItem, Menu, Link } from '@material-ui/core';
import { EventAvailable, SupervisorAccount, PersonAdd, Person } from '@material-ui/icons';